deploy/iso/minikube-iso/package/crictl-bin/crictl-bin.hash

```diff
@@ -1 +1,3 @@
 sha256 ccf83574556793ceb01717dc91c66b70f183c60c2bbec70283939aae8fdef768  crictl-v1.11.1-linux-amd64.tar.gz
+sha256 9bdbea7a2b382494aff2ff014da328a042c5aba9096a7772e57fdf487e5a1d51  crictl-v1.13.0-linux-amd64.tar.gz
+
```

Comment:
trailing newline, please fix and squash the whitespaces fixes into one single commit

Reply:
Thanks for noticing. Done!